includes/theme-compat/header.php

```diff
@@ -33,15 +32,30 @@
             </div>
             <nav class="site-header-navigation">
                 <ul class="menu">
-                    <li class="menu-item current-menu-item">
+                    <li class="menu-item <?php if(isset($page_name) && $page_name == 'home'){ ?> current-menu-item <?php } ?>">
                         <a href="/index.php">Home</a>
                     </li>
+                    <?php if(isset($_SESSION['id_user'])){?>
+                        <?php if(isset($_SESSION['is_admin']) && $_SESSION['is_admin']){?>
+                    <li class="menu-item <?php if(isset($page_name) && $page_name == 'dashboard'){ ?> current-menu-item <?php } ?> ">
```

Comment:
if this is a nested `if` we have the tab so it can be visible that's art of the if statement

Reply:
I forgot to format the code.
Is this the correct way it should look:

`                    <?php if (isset($_SESSION['id_user'])) { ?>
                        <?php if (isset($_SESSION['is_admin']) && $_SESSION['is_admin']) { ?>
                            <li class="menu-item <?php if (isset($page_name) && $page_name == 'dashboard') { ?> current-menu-item <?php } ?> ">
                                <a href="/dashboard.php">Dashboard</a>
                            </li>
                        <?php } ?>
                        <li class="menu-item <?php if (isset($page_name) && $page_name == 'profile') { ?> current-menu-item <?php } ?>">
                            <a href="/profile.php">My Profile</a>
                        </li>
                        <li class="menu-item">
                            <a href="/signout.php">Sign Out</a>
                        </li>
                    <?php } ?>`

Reply:
@yanachkov-devix let's update it then :)